src/vm/nsenums.h

```diff
@@ -73,4 +73,6 @@ DEFINE_NFT(NFT_ILLEGAL,                     1,                      true)
 DEFINE_NFT(NFT_WINDOWSFOUNDATIONIREFERENCE, sizeof(IUnknown*),      true)  // Windows.Foundation.IReference`1 is marshaled to System.Nullable`1.
 #endif // FEATURE_COMINTEROP
 
+DEFINE_NFT(NFT_FIXEDBUFFER,                 0,                      true) // Inline fixed fields
```

Comment:
This should be false instead of true, since FixedBufferAttribute isn't a valid winrt attribute.